contracts/standard/permission/ArbitrablePermissionList.sol

```diff
@@ -263,9 +262,9 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
      */
     function isPermitted(bytes32 _value) public view returns (bool allowed) {
         Item storage item = items[_value];
-        bool _registered = item.status <= ItemStatus.Resubmitted ||
+        bool _offList = item.status <= ItemStatus.Resubmitted ||
```

Comment:
Can we call it `_notRegistered`?

Reply:
`not`'s are confusing in boolean variable names. `off` isn't the best either. What about `_excluded`?

Reply:
Sounds good.